src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/TransactionTests.cs

```diff
@@ -83,7 +83,6 @@ public async Task TransactionalSendRollbackTest(bool partitioned, bool sessionEn
                     using (var ts = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
                     {
                         await sender.SendAsync(message).ConfigureAwait(false);
-                        ts.Dispose();
```

Comment:
If I'm missing something here, please let me know... but it seemed quite useless to call `dispose` the moment before the block scope ends and `dispose` gets called by the `using`.

Reply:
It might have indirectly tested that calling dispose multiple times doesn't throw but I doubt it. Removing seems fine.